### DISCUSSION TITLE
Multi-animal tracking different species

### DISCUSSION THREAD

**Discussion by petroschalas** [talmolab/sleap#900] [Help!]
Hello everyone!

Thank you for a very easy to use and understand pose-estimation software. I am interested in using SLEAP to track two different animals at the same time with much different sizes, a mouse and a cockroach and I was wondering if it is possible to use two different skeletons for tracking each animal. I tried using the same skeleton for both, using the top-down approach, but predictions are not so acurate (I am getting skeletons which include both animals when in close proximity with each other). Do you have any advice on how to improve the tracking in this situation if two different skeletons are not possible ?

Thank you very much for your time,

Petros

**Comment by talmo** [comment]
Hi @petroschalas,

Our recommendation for this type of use case is to train two separate models -- one for each skeleton type (mouse and cockroach) -- and run them separately on the same videos. You'll get two sets of predictions, but you can just load them both during analysis.

Cheers,

Talmo
